src/connectors/articleService.ts

```diff
@@ -965,12 +973,12 @@ export class ArticleService extends BaseService {
     const { hits, ...rest } = await this.meili.index('articles').search(key, {
       limit: take,
       offset: skip,
-      filter: ['state = active'],
+      // filter: ['state = active'],
```

Comment:
this change doesn't relate to article publishing?

Reply:
right; but the filter is for articles searchV1 and no longer needed, could be placed into another PR but...